run_tests.sh

```diff
@@ -121,14 +122,25 @@ function run_tests()
   # amount of time for the integration tests.
   if [[ "${integration_tests_setup}" == 1 ]]; then
     printf '\n' # add new line after the last "OK"
-    say 'Tearing down environment used in integration tests...'
-    teardown_container_environment
+    say 'Tearing down containers used in integration tests...'
+    teardown_containers
     printf '\n'
   fi
 
   report_results "$total" "$success" "$notfound" "$fail" "$test_failure_list"
 }
 
+function clear_unit_tests_cache()
+{
+  say 'Clearing unit tests cache...'
```

Comment:
If there is nothing to clean, just say that; otherwise, you can give the wrong idea to the user. Maybe say, `Skip unit test clean` or `Clear unit test is not necessary` or anything like that.